app/LoadedApp/LoadedApp.tsx

```diff
@@ -301,7 +301,7 @@ class LoadedAppClass extends Component<LoadedAppClassProps, AppStateLoaded> {
   setInfo = (newInfo: InfoType) => {
     // If the price is not set in this object, copy it over from the current object
     const { info } = this.state;
-    if (!!info && !!info.zecPrice && !newInfo.zecPrice) {
+    if (info.zecPrice && !newInfo.zecPrice) {
```

Comment:
Is this just checking to ensure `info.zecPrice` isn't `NaN` (or 0, I guess)?

Reply:
Yo can check the InfoType, which have `zecPrice` as a number, no `undefined` no `null`. This `if (info.zecPrice && !newInfo.zecPrice)` check `zecPrice` like this: if is 0 -> false, and if is not 0 -> true. I changed this because I know `info` exists yes or yes, could be {} but exists, I don;t have to check this out.